bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -243,7 +243,11 @@ public String toString() {
      */
     @Override
     public void stateChanged(Item item, State oldState, State newState) {
+        State oldGroupState = this.state;
         setState(function.calculate(members));
+        if (!oldGroupState.equals(this.state)) {
+            sendGroupStateChangedEvent(item.getName(), this.state, oldGroupState);

```

Comment:
@kaikreuzer : perhaps I have got that wrong?.
We could leave this method completely unimplemented. It would change nothing on the current event behavior because this.state is equal to oldGoupState (updateState() set it before) and so no event is send.  But IMHO an empty method would lead to  an unexpected behavior for users. In this way the method reacts as expected if we change the observer notification in the future.